cognite/replicator/replication.py

```diff
@@ -10,6 +10,13 @@
 from cognite.client.data_classes.raw import Row
 
 
+def contains(list, filter):
+    for x in list:
+        if filter(x):
+            return True
+    return False
```

Comment:
I think this could be replaced with the built-in any function.
```python
any(filter(obj) for obj in list))
```

Reply:
Thanks, nice catch, much better